stable/contour/templates/crb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create -}}
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: {{ template "contour.fullname" . }}
+    chart: {{ template "contour.chart" . }}
+    release: {{ .Release.Name }}
+  name: {{ template "contour.fullname" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: contour
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "contour.fullname" . }}
```

Comment:
create a helper for the serviceaccountname, see: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md and use that for consistency

Reply:
Created helper for service account.